templates/rhsm-katello-reconfigure.erb

```diff
@@ -71,9 +71,14 @@ if [ -d $CA_TRUST_ANCHORS ]; then
   cp $KATELLO_CERT_DIR/$KATELLO_CERT $CA_TRUST_ANCHORS
   update-ca-trust
 
-  #restart if docker service is installed
-  service docker status >/dev/null && \
+  # restart docker if it is installed and running
+  if [ -f /usr/lib/systemd/system/docker.service ]; then
```

Comment:
@elyezer, I totally forgot about checking for docker.service in particular, so my example code is incomplete. Whoops! Still, I think there's a case to be made for using utilities to check for the presence of unit files or service files instead of directly querying the filesystem for the presence of files. I'd love to hear what others have to say.